helm/metac/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: metac
+apiVersion: v1
+appVersion: "v0.1.0"
+description: A Helm chart for Metac
+version: 0.1.0
```

Comment:
I guess this is the version I was talking about. If for example, there is a bug in helm charts or some improvements in helm charts keeping Metac version same. How do we follow the version changes in helm charts?

Will it be good to have only major & minor versions updated in Metac while Helm follows Metacmajor.Metacminor.Helmpatch for its version?

Reply:
`appVersion` is just metada and refers to the version of the app the chart is installing, `metac` in this case. It only used for indexing the chart in the repo and display purposes. `version` is the version of the hel chart, they are pretty much independent but you can choose to keep them in sync if you want.